tikv-migration/tikv-migration-verify-pipeline.yaml

```diff
@@ -29,3 +29,21 @@ tasks:
       credentials:
         - jenkinsID: codecov-token-tikv-migration
           key: TIKV_MIGRATION_CODECOV_TOKEN
+
+    - taskName: "cdc-it" 
+      checkerName: "it-check" 
```

Comment:
There may not be such a checker. Suggest to ask @purelind for help.

Reply:
@purelind recommends using comon_check.